bin/setup

```diff
@@ -30,47 +22,56 @@ end.parse!
 # path to your application root.
 APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
 
-Dir.chdir APP_ROOT do
-  puts "== Installing dependencies =="
+def system!(*args)
+  system(*args) || abort("\n== Command #{args} failed ==")
+end
+
+chdir APP_ROOT do
```

Comment:
Yeah, I thought that was odd too when I first saw it.  I'd like to minimize our changes on top of the rails template though so I'm ok with it.